metricbeat/internal/metrics/cpu/metrics.go

```diff
@@ -91,7 +104,18 @@ func (m *Monitor) Fetch() (Metrics, error) {
 	oldLastSample := m.lastSample
 	m.lastSample = metric
 
-	return Metrics{previousSample: oldLastSample.totals, currentSample: metric.totals, count: len(metric.list), isTotals: true}, nil
+	// There isn't a 'total' for the CPU/Core frequency, so we average all the
```

Comment:
Why not report all of the values in separate metrics?

Reply:
> You might want to consider looking at how the system/core metricset does things, that might be a better way to report a lot of this.

I was looking at it, and what it does is to get the aggregated metrics provided by the OS (at least on Linux), `/proc/stat` has got an aggregated line there. It really makes things easy.

> It's important to remember that CPU data that comes from sources like /proc/cpuinfo can be weirdly heterogeneous, particularly on multi-socket systems and VMs, I would strongly advise against trying to "sum" things.

Indeed. I think I'll just keep them out of `system/cpu`. At least for now.
It feels pretty odd to introduce this 'core' concept into `system/cpu`.